playwright-tests/tests/profile.spec.js

```diff
@@ -0,0 +1,176 @@
+import { expect, test } from "@playwright/test";
+import { ROOT_SRC } from "../util/constants";
+import path from "path";
+
+test.describe("?page=profile", () => {
+  test.beforeEach(async ({ page }) => {
+    await page.goto(`/${ROOT_SRC}?page=profile`);
+  });
+
+  test.describe("User logged in", () => {
+    test.beforeEach(async ({ page }) => {
+      // Intercept IPFS requests
+      await page.route("**/add", async (route) => {
```

Comment:
Since this is only used by the one test, we should move it there:

"Should navigate to Edit Profile and save correct data"